streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -274,6 +287,7 @@ public void cleanup() throws IOException {
             task.closeDirty();
             task = null;
         }
+        mockito.finishMocking();
```

Comment:
Could you put this after `Utils.delete(BASE_DIR)` to ensure that the directory is deleted also if mockito throws an exception when a stub is unnecessary?